Automattic/wp-calypso
# [Odie] QoL changes

## Proposed Changes

- More code clean-ups
- Add logger via DI

## Testing Instructions

Please, test the Wapuu flow:
1. Check that you can still ask for help
2. Check that you can still ask questions
3. Check that you can still give feedback
4. Check that you can still init new chats (clear chat)
5. Check that events are still triggered:

- calypso_odie_chat_cleared
- calypso_odie_chat_message_action_click
- calypso_odie_chat_message_action_sources
- calypso_odie_chat_jump_to_recent_click
- calypso_odie_chat_message_action_feedback
- calypso_odie_chat_message_action_send
- calypso_odie_chat_message_action_receive
- calypso_odie_chat_message_error
- calypso_odie_extra_contact_option

## Pre-merge Checklist
